test/spot/public.test.ts

```diff
@@ -0,0 +1,57 @@
+import {SpotClient} from "../../src/spot-client";
+import {successResponseListSpot, successResponseObjectSpot} from "../response.util";
+import "axios-debug-log"
+
+describe('Spot REST API Endpoints', () => {
+    const useLivenet = true;
+
+    const KEY = process.env.BYBIT_KEY
+    const SECRET = process.env.BYBIT_SECRET
+    const api = new SpotClient(undefined, undefined, useLivenet, {disable_time_sync: true});
+
+    const symbol = 'BTCUSDT';
+    const interval = '15';
+    const timestampOneHourAgo = (new Date().getTime() / 1000) - (1000 * 60 * 60);
+    const from = Number(timestampOneHourAgo.toFixed(0));
+
+    describe('Spot Public endpoints', () => {
+
+        it('getServerTime()', async () => {
+            expect(await api.getServerTime()).toMatchObject(successResponseObjectSpot(""));
+        });
+
+        it('getSymbols()', async () => {
+            expect(await api.getSymbols()).toMatchObject(successResponseListSpot(""));
+        });
+
+        it('getOrderBook()', async () => {
+            expect(await api.getOrderBook(symbol)).toMatchObject(successResponseObjectSpot());
+        });
```

Comment:
Looks great! key and secret aren't being used yet but I guess it's just public endpoints at first anyway. Please just fix:
- remove the axios debug log dependency & import
- update the indent (2 spaces)
- add semi colons where missing

Reply:
updated the indentation. You should probably use prettier + husky for code style consistency

Reply:
Definitely - haven't gotten to setting it up yet